From 869ee62416fc01aaf86f4540efcf262ac639b24c Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Tue, 30 May 2023 09:38:38 +0200
Subject: [PATCH 0384/1141] dt-bindings: pci: stm32: add pme optional interrupt

The synopsys controller integrated in STM32MP25 can manage the PME MSIs through
platform interrupt lines connected to the GIC. Hence describe this interrupt.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I9a0a919e683110f320a120a142db8d41ca94ed95
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/309513
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 .../devicetree/bindings/pci/st,stm32-pcie-host.yaml   | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/Documentation/devicetree/bindings/pci/st,stm32-pcie-host.yaml b/Documentation/devicetree/bindings/pci/st,stm32-pcie-host.yaml
index b0e562fad5b2..57688866f2f5 100644
--- a/Documentation/devicetree/bindings/pci/st,stm32-pcie-host.yaml
+++ b/Documentation/devicetree/bindings/pci/st,stm32-pcie-host.yaml
@@ -47,12 +47,14 @@ properties:
     const: core
 
   interrupts:
-    description: AER interrupts
-    maxItems: 1
+    description: AER and PME interrupts
+    minItems: 1
+    maxItems: 2
 
   interrupt-names:
     items:
       - const: aer_msi
+      - const: pme_msi
 
   phys:
     maxItems: 1
@@ -119,8 +121,9 @@ examples:
                              <0 0 0 2 &intc 0 0 GIC_SPI 265 IRQ_TYPE_LEVEL_HIGH>,
                              <0 0 0 3 &intc 0 0 GIC_SPI 266 IRQ_TYPE_LEVEL_HIGH>,
                              <0 0 0 4 &intc 0 0 GIC_SPI 267 IRQ_TYPE_LEVEL_HIGH>;
-             interrupts = <GIC_SPI 232 IRQ_TYPE_LEVEL_HIGH>;
-             interrupt-names = "aer_msi";
+             interrupts = <GIC_SPI 232 IRQ_TYPE_LEVEL_HIGH>,
+                          <GIC_SPI 238 IRQ_TYPE_LEVEL_HIGH>;
+             interrupt-names = "aer_msi", "pme_msi";
              #address-cells = <3>;
              #size-cells = <2>;
              ranges = <0x01000000 0 0x10010000 0x10010000 0 0x10000>,
-- 
2.39.2

